tcs-service/src/test/java/com/transformuk/hee/tis/tcs/service/api/PostResource2Test.java

```diff
@@ -314,6 +315,20 @@ public void getPostPlacementShouldReturnUnauthWhenUserNotPartOfSameTrustNew() th
     verify(placementSummaryDecorator, never()).decorate(any());
   }
 
+  @Test
+  public void deletePostShouldReturnUnauthWhenUserDbNotSameAsPostOwner() throws Exception {
+    long postId = 1L;
+
+    doThrow(new AccessUnauthorisedException("")).when(postService)
+        .checkTheLoggedInUserDbSameAsPostOwner(eq(postId), any(UserProfile.class));
```

Comment:
If you accept `any(UserProfile.class)`, then you wouldn't need to invoke `mockUserprofile()` for this test.  Is it done for some other tests?  If I remember correctly, you only needed to add one for an integration test.